pipettes/firmware/hardware_config.c

```diff
@@ -33,23 +33,45 @@ static PipetteHardwarePin get_gpio_ht(PipetteHardwareDevice device) {
     }
 }
 
-IRQn_Type get_interrupt_line(const PipetteType pipette_type) {
-    switch (pipette_type) {
-        case NINETY_SIX_CHANNEL:
-        case THREE_EIGHTY_FOUR_CHANNEL:
+
+static IRQn_Type get_interrupt_line_lt(PipetteGPIOInterrupt gpio_pin_type) {
+    switch (gpio_pin_type) {
+        case pipette_gpio_data_ready:
+            // Tip sense External interrupt line 3
+            return EXTI3_IRQn;
+        case pipette_gpio_tip_sense:
+        default:
+            // Tip sense External interrupt line 2
+            return EXTI2_IRQn;
+    }
+}
+
+static IRQn_Type get_interrupt_line_ht(PipetteGPIOInterrupt gpio_pin_type) {
+    switch (gpio_pin_type) {
+        case pipette_gpio_data_ready:
             // External interrupt lines 15:10
-            // TODO(lc: 10-20-2022) This will only
-            // return the front sensor board data ready line
-            // for now. We need to support and set-up both
-            // for the 96-chan and 8-chan
             // PC6 -> GPIO_EXTI6 (EXTI9_5_IRQn)
             // PC11 -> GPIO_EXTI11 (EXTI15_10_IRQn)
-            return EXTI15_10_IRQn;
+            return EXTI15_10_IRQn | EXTI9_5_IRQn;
+        case pipette_gpio_tip_sense:
+        default:
+            // tip sense
+            // PC7 -> GPIO_EXTI6 (EXTI9_5_IRQn)
+            // PC12 -> GPIO_EXTI11 (EXTI15_10_IRQn)
+            return EXTI15_10_IRQn | EXTI9_5_IRQn;
```

Comment:
I don't think that this is going to give you the behavior you want. The `IRQn` enums are not single-bit values, but instead indices into the NVIC bitfields. Returning two IRQn values OR'd together is probably going to end up enabling a completely different IRQ.